test/workerinfo_test.js

```diff
@@ -117,13 +117,13 @@ suite('provisioners and worker-types', () => {
         name: 'kill',
         title: 'Kill Provisioner',
         context: 'provisioner',
-        url: 'https://hardware-provisioner.mozilla-releng.net/v1/power-cycle',
+        url: 'https://hardware-provisioner.mozilla-releng.net/v1/power-cycle/:provisionerId',
         description: 'Remove provisioner prov-B',
       }, {
         name: 'kill',
         title: 'Kill Worker Type',
         context: 'worker-type',
-        url: 'https://hardware-provisioner.mozilla-releng.net/v1/power-cycle',
+        url: 'https://hardware-provisioner.mozilla-releng.net/v1/power-cycle/:provisionerId/:workerType',
```

Comment:
Against please consider `<provisionerId>` instead of `:provisionerId` as the parameter format... this is more in line with what we do in `references.json`..